ports/gflags/common.mk

```diff
@@ -0,0 +1,66 @@
+ifndef QCONFIG
+QCONFIG=qconfig.mk
+endif
+include $(QCONFIG)
+
+NAME=gflags
+
+DIST_BASE=$(PRODUCT_ROOT)/../../../clapack
```

Comment:
Wrong path?

Reply:
Yeah - I must have missed it since I always explicitly set QNX_PROJECT_ROOT. Fixed with 06b46d71a60257eef595cbe883752dd44ddd476a !